src/devices/openxrheadset/OpenXrInterface.cpp

```diff
@@ -735,6 +762,31 @@ bool OpenXrInterface::prepareXrActions()
     return true;
 }
 
+bool OpenXrInterface::prepareHandTracking() {
+
+    if (!m_pimpl->hand_tracking_supported) {
+        return true;
+    }
+    XrHandTrackerCreateInfoEXT handTrackerCreateInfo =
+    {
+        .type = XR_TYPE_HAND_TRACKER_CREATE_INFO_EXT,
+        .next = NULL,
+        .hand = XR_HAND_LEFT_EXT,
+        .handJointSet = XR_HAND_JOINT_SET_DEFAULT_EXT
+    };
+
+    XrResult result = m_pimpl->pfn_xrCreateHandTrackerEXT(m_pimpl->session, &handTrackerCreateInfo, &m_pimpl->left_hand_tracker);
+    if (!m_pimpl->checkXrOutput(result, "Failed to create left hand tracker"))
+        return false;
+
+    handTrackerCreateInfo.hand = XR_HAND_RIGHT_EXT;
+    result = m_pimpl->pfn_xrCreateHandTrackerEXT(m_pimpl->session, &handTrackerCreateInfo, &m_pimpl->right_hand_tracker);
+    if (!m_pimpl->checkXrOutput(result, "Failed to create right hand tracker"))
+        return false;
```

Comment:
Here instead of returning ``false``, you could print the error and set ``m_pimpl->hand_tracking_supported`` to false. This could fix the test, where the extension is supported by the runtime, but it cannot be used since there is no headset connected.